fire/ident.py

```diff
@@ -14,7 +14,7 @@
 KØBSTADSNUMMERMØNSTER = re.compile("^[Kk][ ]*-[0-9]*-[0-9]*$")
 "Generaliseret mønster for købstadsnumre"
 
-VANDSTANDSBRÆTMØNSTER = re.compile(r"^([Kk][ ]*|[0-9]*)-[0-9]*-V\.[0-9]*$")
+VANDSTANDSBRÆTMØNSTER = re.compile(r"^([Kk][ ]*|[0-9]*)-[0-9]*-[Vv]\.[0-9]*$")
```

Comment:
Du har muligvis overvejet det, men bare til info, så kan det regulære udtryk holdes mere simpelt ved at skrive det sådan og kompilere med flaget `re.I`/`re.IGNORECASE`:

```suggestion
VANDSTANDSBRÆTMØNSTER = re.compile(r"^(K[ ]*|[0-9]*)-[0-9]*-V\.[0-9]*$", flags=re.I)
```

* https://docs.python.org/3/library/re.html#re.I
* https://docs.python.org/3/library/re.html#re.compile

Du bestemmer;)